src/servers/graphql-old-server.ts

```diff
@@ -263,7 +266,14 @@ const resolvers = {
         return lnInvoice.paymentRequest
       },
       // FIXME: move to query
-      updatePendingInvoice: async ({ hash }) => wallet.updatePendingInvoice({ hash }),
+      updatePendingInvoice: async ({ hash }) => {
```

Comment:
`Todo` manually confirm nothing breaks in the existing calls to this endpoint ([see comment](https://github.com/GaloyMoney/galoy/pull/509/files#r698644968))